modules/packages/vendored/yum.mustache

```diff
@@ -87,7 +87,16 @@ def get_package_data():
         # Absolute file.
         sys.stdout.write("PackageType=file\n")
         sys.stdout.flush()
-        return subprocess_call([rpm_cmd, "--qf", rpm_output_format, "-qp", pkg_string])
+        process = subprocess_Popen([rpm_cmd, "--qf", rpm_output_format, "-qp", pkg_string], stdout=subprocess.PIPE)
+        (stdoutdata, _) = process.communicate()
+
+        if process.returncode != 0:
+            return process.returncode
+
+        for line in stdoutdata.decode("utf-8").splitlines():
+            sys.stdout.write(line.replace("(none):","") + "\n")
```

Comment:
```suggestion
            sys.stdout.write(line.replace("(none):", "") + "\n")
```

Reply:
yeah, ran black on it as a separate commit now :+1: